ui/components/multichain/asset-picker-amount/index.scss

```diff
@@ -0,0 +1,25 @@
+.asset-picker-amount {
+  &__box {
+    display: flex;
+    align-items: center;
+    margin-top: 3px;
+    margin-bottom: 1px;
+    background-color: var(--color-background-default);
+    border: 2px solid var(--color-primary-default);
+    padding: 0 12px 0 12px;
```

Comment:
Should be able to use utility props for all these styles with the exception of the margin-top and margin-bottom but I think it's likely that we want to use something else here. We use a 4px grid so any values outside of a multiple of 4 are UI smells 😅